core/drivers/clk/sam/at91_utmi.c

```diff
@@ -136,3 +137,59 @@ at91_clk_register_utmi(struct pmc_data *pmc, const char *name,
 
 	return clk;
 }
+
+struct clk *at91_clk_register_utmi(struct pmc_data *pmc,
+				   const char *name,
+				   struct clk *parent)
+{
+	return at91_clk_register_utmi_internal(pmc, name, &utmi_ops, parent);
+}
+
+#ifdef CFG_SAMA7G5
```

Comment:
Remove this #ifdef too. We want as few compile-time conditionals as possible (for simplicity and code coverage reasons).

Reply:
`#ifdef` removed, thanks.